src/common/latest/helpers/constants.ts

```diff
@@ -36,13 +38,16 @@ export enum AzureBuildVariables {
 }
 
 export enum TaskVariables {
-  SonarQubeServerVersion = "SONARQUBE_SERVER_VERSION",
-  SonarQubeScannerMode = "SONARQUBE_SCANNER_MODE",
-  SonarQubeScannerParams = "SONARQUBE_SCANNER_PARAMS",
-  SonarQubeScannerReportTaskFile = "SONARQUBE_SCANNER_REPORTTASKFILE",
-  SonarQubeEndpoint = "SONARQUBE_ENDPOINT",
-  SonarQubeScannerMSBuildExe = "SONARQUBE_SCANNER_MSBUILD_EXE",
-  SonarQubeScannerMSBuildDll = "SONARQUBE_SCANNER_MSBUILD_DLL",
+  SonarServerVersion = "SONAR_SERVER_VERSION",
+  SonarScannerMode = "SONAR_SCANNER_MODE",
+  SonarScannerParams = "SONAR_SCANNER_PARAMS",
```

Comment:
The real equivalent of `SONARQUBE_SCANNER_PARAMS` is `SONAR_SCANNER_JSON_PARAMS` ([bootstrap spec](https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/3155001395/Scanner+Bootstrappers+implementation+guidelines)), do we want to move this to the new name to keep the same behavior?

I think it's used to make sure MSBuild scanner gets the sonar props (in this case there's no analyze task), not 100% sure about this
```suggestion
  SonarScannerJsonParams = "SONAR_SCANNER_JSON_PARAMS",
```

Reply:
sure, makes sense to me to align the scanners across the board